rfcs/tag-component.md

```diff
@@ -0,0 +1,96 @@
+# Tag Component
+
+### Exports
+
+- `StatefulTag`
```

Comment:
I'm curious what's your use case here. Why do you want to have states in a Tag.

Reply:
+1. Seems like a tag should be stateless.

Reply:
removed